pyndemic/controller.py

```diff
@@ -68,22 +69,37 @@ def _flush_signals(self):
 
 
 class GameController(AbstractController):
-    def __init__(self, random_state=None):
+    def __init__(self, **settings):
         super().__init__()
 
         self.game = None
         self.characters = {}
         self.current_character = None
         self.name_cycle = None
         self._loop = None
-        self.random_state = random_state
+        self.settings = config.get_settings()
+
+        if settings:
+            self.setup(settings)
+
+    def setup(self, settings):
+        manual_settings = self.settings['Other']
+
+        for key, value in settings.items():
+            if key == 'players':
+                value = ' '.join(value)
+            if value is not None:
+                manual_settings[key] = str(value)
+
+        if 'players' not in manual_settings:
+            manual_settings['players'] = 'Alpha Bravo Charlie Delta'
```

Comment:
It's unclear how your set roles for players, which person will be Doctor, Researcher etc
Does it depend only on order in `manual_settings['players']` list?

Reply:
Such roles are not provided in the game for now.
Supporting of unique characters (Doctor etc.) is the question for next releases.